### ISSUE TITLE
Using keyboard to skip backwards skips forwards instead

### ISSUE THREAD

**Issue by davidvidmar** [f1multiviewer/issue-tracker#309]
When trying to review a stream using keyboard, both keyboard shortcuts left and right jump forwards, docs state that left arrow should skip backwards.

I spotted this on v1.11.5 but I'm sure this is not the first version that has mentioned issue as I just updated. Running on MacOS.

**Comment by f1multiviewer** [comment]
Correct, they don't work during live, I noticed this too. I'll give it a look!

**Comment by Zixim** [comment]
**, (comma) Jump backwards by 1 frame** and **. (period) Jump forwards by 1 frame** are also non-functional :/
